etw/rust/tracelogging/src/native.rs

```diff
@@ -69,10 +69,14 @@ impl ProviderContext {
         {
             result = 50; // ERROR_NOT_SUPPORTED
         }
-        #[cfg(all(windows, feature = "etw"))]
+        #[cfg(all(windows, feature = "etw", feature = "user_mode"))]
```

Comment:
Instead of feature = user_mode, can you use `not(feature = "kernel_mode")`?

Reply:
Replaced user_mode with not(feature="kernel_mode")